docs/api.md

```diff
@@ -7,32 +7,40 @@ permalink: /docs/api/
 ##Compiling a template
 Assuming you have the dust compiler in your JS environment, you can compile a Dust template source file to JavaScript form. The following will compile this very simple template "Hello {name}!" to a JavaScript string that will also register it as a template named "intro".
 
-	var compiledTemplate = dust.compile("Hello {name}!", "intro");
+```
+var compiledTemplate = dust.compile("Hello {name}!", "intro");
```

Comment:
This file should look like API documentation instead of tutorial documentation. The `dustc` file is closer to what we need. I think that examples are useful, but we need a definition of each method signature and what the output of that method is.